pkg-manager/plugin-commands-installation/test/install.ts

```diff
@@ -52,3 +53,126 @@ test('install with no store integrity validation', async () => {
 
   expect(fs.readFileSync('node_modules/is-positive/readme.md', 'utf8')).toBe('modified')
 })
+
+describe('install injected dependencies', () => {
+  for (const {
+    title,
+    withLockfile,
+  } of [
+      {
+        title: 'with shared lockfile',
+        withLockfile: true,
+      }, {
+        title: 'with individual lockfiles',
+        withLockfile: false,
+      },
+    ]) {
```

Comment:
jest has built-in way to loop over cases

https://jestjs.io/blog/2018/05/29/jest-23-blazing-fast-delightful-testing#jest-each

Reply:
Updated